### PR TITLE
run after_install hooks

### PR DESCRIPTION
**Pull request by anthonybishopric**

### CODE REVIEW

File: pkg/preparer/orchestrate.go [hunk 1]
```diff
@@ -213,6 +213,14 @@ func (p *Preparer) installAndLaunchPod(newManifest *pods.PodManifest, pod Pod, l
 			return false
 		}
 
+		err = p.hooks.RunHookType(hooks.AFTER_INSTALL, pod, newManifest)
+		if err != nil {
+			logger.WithFields(logrus.Fields{
+				"err":   err,
+				"hooks": hooks.AFTER_LAUNCH,
```
Comment (anthonybishopric) [inline pkg/preparer/orchestrate.go]: woops, fixing

File: pkg/preparer/orchestrate.go [hunk 2]
```diff
@@ -200,7 +200,7 @@ func (p *Preparer) installAndLaunchPod(newManifest *pods.PodManifest, pod Pod, l
 		if err != nil {
 			logger.WithFields(logrus.Fields{
 				"err":   err,
-				"hooks": "before_install",
+				"hooks": hooks.BEFORE_INSTALL,
```
Comment (petertseng) [inline pkg/preparer/orchestrate.go]: you are changing this but not the "after_launch"?

Comment (anthonybishopric) [inline pkg/preparer/orchestrate.go]: just fixed that, good catch

**Comment by petertseng** [comment]
:+1: 
(Y)

**Comment by petertseng** [comment]
That's weird that the CI build fails. It works locally!

**Pull request merged**